central/main.go

```diff
@@ -815,6 +816,12 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 			ServerHandler: clusterCveCsv.ClusterCVECSVHandler(),
 			Compression:   true,
 		},
+		{
+			Route:         "/api/policy/custom-resource/save-as-zip",
```

Comment:
Should this be `policy` or `policies`?

Reply:
I think the convention in the list above is to use the singular

Reply:
ah, I see now that this is mounted at a different top-level path, `api` vs `v1`.  I think the point stil stands regarding convention, but it seems clear we are not trying to make it look like part of the existing `/v1/policies` API.